pyvista/plotting/charts.py

```diff
@@ -4459,6 +4468,6 @@ def __iter__(self):
         for chart in self._charts:
             yield chart
 
-    def __del__(self):  # pragma: no cover
+    def __del__(self):
```

Comment:
This is how this all started. I naively added a `__del__` when fixing https://github.com/pyvista/pyvista/pull/2991, only to realise that this is never called. Had to skip this for coverage then, since you can't call `__del__` on an object that refuses to die.